src/main/java/com/zendesk/maxwell/schema/AbstractSchemaStore.java

```diff
@@ -0,0 +1,56 @@
+package com.zendesk.maxwell.schema;
+
+import java.sql.SQLException;
+import java.sql.Connection;
+import java.util.List;
+import java.util.ArrayList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.zendesk.maxwell.MaxwellContext;
+import com.zendesk.maxwell.BinlogPosition;
+import com.zendesk.maxwell.schema.ddl.SchemaChange;
+import com.zendesk.maxwell.schema.ddl.ResolvedSchemaChange;
+import com.zendesk.maxwell.schema.ddl.InvalidSchemaError;
+
+public abstract class AbstractSchemaStore {
+	static final Logger LOGGER = LoggerFactory.getLogger(AbstractSchemaStore.class);
+	protected final MaxwellContext context;
+
+	protected AbstractSchemaStore(MaxwellContext context) {
+		this.context = context;
+	}
+
+	protected Schema captureSchema() throws SQLException {
+		try(Connection connection = context.getReplicationConnection()) {
+			LOGGER.info("Maxwell is capturing initial schema");
```

Comment:
you mean just the log line?  No, I think if we had a `KafkaSchemaStore` it'd want all this code too.